recipes/taxadb/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "taxadb" %}
+{% set version = "0.12.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```

Comment:
Why the `name[0]`, is that common?

Reply:
This was suggested by the conda skeleton script. It is probably okay either way so happy to go with your preference